lib.ml

```diff
@@ -56,7 +62,7 @@ and row_values db = function
 
 type primitive =
 [ Value.t
-| `PFun of Value.t list -> Value.t ]
+| `PFun of Value.t list -> requestData -> Value.t ]
```

Comment:
Nitpick: adding the request_data argument as the first argument might preserve more of the subsequent code (not a big deal though)